src/index.css

```diff
@@ -27,3 +27,9 @@ a {
 .MuiList-padding {
   padding: 0 !important;
 }
+
+.MuiButton-outlined {
+  height: 25px;
+  width: 40px;
+  font-size: 14px !important;
+}
```

Comment:
Why do you write this CSS at main CSS file? This will affect all MuiButton-outlined because it is not custom selector. Consider renaming it or remove to its own CSS file